src/main/resources/fabric.mod.json

```diff
@@ -3,15 +3,15 @@
   "id": "mcde",
   "version": "${version}",
 
-  "name": "Minecraft Dungeons: Enchanting",
+  "name": "MC Dungeons: Enchanting",
   "description": "Brings the Enchantment system from Minecraft Dungeons to Minecraft",
   "authors": [
     "BackupCup",
     "Iamnotagenius"
   ],
   "contact": {
-    "curseforge": "https://legacy.curseforge.com/minecraft/mc-mods/mc-dungeons-enchantments",
-    "modrinth": "https://modrinth.com/mod/mc-dungeons-enchantments",
+    "curseforge": "https://curseforge.com/minecraft/mc-mods/mc-dungeons-enchanting",
```

Comment:
Whilst not necessary, if you are not directing to `legacy.curseforge.com`, use `www.` as the prefix as screen readers rely on this being present to indicate to visually impair individuals that they're looking at a link if the `http(s)` gets skipped for whatever reason